repository.json

```diff
@@ -678,6 +678,34 @@
             "version": "1.9.7",
             "url": "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-1.9.7-windows.zip",
             "fileMatchInside": ".*/phantomjs.exe$"
+        },
+        {
+            "name": "marionette",
```

Comment:
Should't we call the driver geckodriver instead of marionette? According the geckodriver github page Marionette is the protocol name while the drivers name is geckodriver.